frontend/src/components/Chat.jsx

```diff
@@ -46,12 +46,14 @@ class Chat extends React.Component {
 
   sendMessage (e) {
     e.preventDefault()
-    if (this.state.chatInputValue) {
+    if (this.state.chatInputValue && this.state.chatInputValue.length <= 128) {
       this.props.socket.emit('chat', {
         name: this.props.playerName,
         message: this.state.chatInputValue
       })
       this.setState({ chatInputValue: '' })
+    } else {
```

Comment:
You technically don't need this here. You can just move 
```
this.setState({ chatInputValue: '' })
```
out of the if statement. Just a minor critique which can reduce code.